lib/build/checkFiles.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
```

Comment:
I think our standard for file naming is hyphens, so `check-files.js` (sorry, not a comment on this line, not sure where to put it)

Reply:
Okay, we'll have to go through and rename a bunch of files then, I will make that change in a separate commit.

Reply:
Ah, I did know that, thanks for catching it.